app/views/home/index.html.erb

```diff
@@ -1,2 +1,80 @@
-<h1>Welcome to flextensions</h1>
-<p>Team Members: <%= @team_members.join(", ") %></p>
\ No newline at end of file
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN"
+    "http://www.w3.org/TR/html4/strict.dtd">
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Flextensions">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-blue.min.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/style.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/flash.css">
+    <script src="/static/js/utils.js"></script>
+    
+    <title>Flextensions Landing</title>
+  </head>
+
+  <body>
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
+      <header class="mdl-layout__header">
+        <div class="mdl-layout__header-row">
+        
+          
+            <a href="/">
+              <span class="hidden-small mdl-layout__title">
+                
+              </span>
+            </a>
+          
+          
+          <div class="hidden-small mdl-layout-spacer"></div>
+
+          

```

Comment:
(and stuff below...)

The navbar setup belongs in a shared partial, and this should use bootstrap's classes. 

While we're modeling the flow of the seating app, we don't want to keep all the material UI bits.